### PR TITLE
Remove Tenant columns anonymisation

### LINKED ISSUE #2373: [FlexibleBackend] Remove columns anonymisation
## Context

Currently column names are anonymised and can't be easily read without the associated metadata. This was done to ensure renaming an objectMetadata or fieldMetadata wouldn't imply a DB migration.
In practice, this kind of migration will be rare and we don't expect breaking changes coming from it. 
By removing this logic we can imagine giving access to a workspace schema so that tenant users can run SQL directly inside (we need to make sure those requests are scoped to their tenant schema only ofc)

### PR DESCRIPTION
**Pull request by Weiko**
Fix https://github.com/twentyhq/twenty/issues/2373

### CODE REVIEW

File: server/src/metadata/field-metadata/utils/field-metadata.util.ts [hunk 1]
```diff
@@ -31,7 +18,11 @@ export function generateColumnName(name: string): string {
  */
 export function generateTargetColumnMap(
   type: FieldMetadataType,
+  isCustomObject: boolean,
+  fieldName: string,
 ): FieldMetadataTargetColumnMap {
+  const columnName = isCustomObject ? fieldName : `_${fieldName}`;
```
Comment (charlesBochet) [inline server/src/metadata/field-metadata/utils/field-metadata.util.ts:24]: it should be the opposite, right?

File: server/src/metadata/field-metadata/utils/field-metadata.util.spec.ts [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+import {
+  generateTargetColumnMap,
+  convertFieldMetadataToColumnActions,
+} from './field-metadata.util';
+
+describe('generateTargetColumnMap', () => {
+  it('should generate a target column map for a given type', () => {
+    const textMap = generateTargetColumnMap(
+      FieldMetadataType.TEXT,
+      true,
```
Comment (charlesBochet) [inline server/src/metadata/field-metadata/utils/field-metadata.util.spec.ts:12]: I would expect _name with customObject set to true

Comment (Weiko) [reply server/src/metadata/field-metadata/utils/field-metadata.util.spec.ts:12]: If it's a custom object we don't need to specify that's a custom field you are trying to add. You only want the _ if it's for a standard object

Comment (Weiko) [reply server/src/metadata/field-metadata/utils/field-metadata.util.spec.ts:12]: Discussed offline, let's put "_" for every customObject

File: server/src/metadata/field-metadata/utils/field-metadata.util.ts [hunk 3]
```diff
@@ -12,16 +9,6 @@ import {
   TenantMigrationColumnActionType,
 } from 'src/metadata/tenant-migration/tenant-migration.entity';
 
-/**
- * Generate a column name from a field name removing unsupported characters.
- *
- * @param name string
- * @returns string
- */
-export function generateColumnName(name: string): string {
```
Comment (Weiko) [inline server/src/metadata/field-metadata/utils/field-metadata.util.ts:21]: This was not used

**Review by charlesBochet — APPROVED** [review]
LGTM

**Pull request merged**